### PR TITLE
Fixed ValueMap for Ensure

### LINKED ISSUE #94: Check the Ensure ValueMap for all resources
#### Details of the scenario you tried and the problem that is occurring
Most resources have the Ensure parameter. Some only allow the value Present, but the ValueMap still contains Absent. Should Absent be accepted if the description states that only Present is allowed?

### PR DESCRIPTION
**Pull request by NikCharlebois**
#### Pull Request (PR) description
For resources that threw an error if Ensure = "Absent" was set, we simply removed Absent from the value map.

#### This Pull Request (PR) fixes the following issues
- Fixes #94

---
This change is [Reviewable](https://reviewable.io/reviews/microsoft/office365dsc/139)

### CODE REVIEW

**Comment by NikCharlebois** [comment]
@ykuijs All yours to review sir!

File: Modules/Office365DSC/DSCResources/MSFT_O365AdminAuditLogConfig/MSFT_O365AdminAuditLogConfig.psm1 [hunk 1]
```diff
@@ -10,8 +10,9 @@ function Get-TargetResource
         $IsSingleInstance,
 
         [Parameter()]
-        [ValidateSet('Present', 'Absent')]
-        [string]$Ensure = 'Present',
+        [ValidateSet('Present')]
+        [string]
```
Comment (ykuijs) [inline Modules/Office365DSC/DSCResources/MSFT_O365AdminAuditLogConfig/MSFT_O365AdminAuditLogConfig.psm1]: In the Set method you updated the type to System.String. Lets do it here and in the Test method as well to keep it consistent.

File: Modules/Office365DSC/DSCResources/MSFT_O365AdminAuditLogConfig/MSFT_O365AdminAuditLogConfig.psm1 [hunk 2]
```diff
@@ -140,8 +132,9 @@ function Test-TargetResource
...
```
Comment (ykuijs) [inline Modules/Office365DSC/DSCResources/MSFT_O365AdminAuditLogConfig/MSFT_O365AdminAuditLogConfig.psm1]: Same as before

**Comment by NikCharlebois** [comment]
@ykuijs Requested Changes Done

**Pull request merged**